python/cudf/cudf/io/orc.py

```diff
@@ -171,7 +171,7 @@ def read_orc_statistics(
     files_statistics = []
     stripes_statistics = []
     for source in filepaths_or_buffers:
-        filepath_or_buffer, compression = ioutils.get_filepath_or_buffer(
+        (filepath_or_buffer, compression,) = ioutils.get_reader_path_or_buf(
```

Comment:
😅 This is what I was originally proposing:
```suggestion
        path_or_buf, compression = ioutils.get_reader_filepath_or_buffer(
```